src/platforms/extensions/Directory.Build.props

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build">
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
   </PropertyGroup>
+
+  <!-- Disable code paths that require project.assets.json files to be present or to be computed. -->
```

Comment:
Did you need these?  They all look netcoreapp specific, I wouldn't expect them to be causing a problem for netstandard build.

Reply:
Yes to disable the Target that tries to resolve stuff from `project.assets.json`, without it I get the following error:

> E:\code\standard\.dotnet\sdk\2.1.401\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets(129,5): error NETSDK1004: Assets file 'E:\code\standard\artifacts\obj\System.CodeDom\project.assets.json' not found. Run a NuGet package restore to generate this file. [E:\code\standard\src\platforms\extensions\System.CodeDom.csproj]

Reply:
I see we're doing this elsewhere: https://github.com/dotnet/standard/search?q=GenerateDependencyFile&unscoped_q=GenerateDependencyFile.  Can we centralize it?

Reply:
Please consider a followup PR to centralize these properties.

Reply:
Lol. One of my follow-up PRs would have been to bring back project restore so that we don't need these hacks.

Reply:
Even better 👍